docs/how-to-guides/develop-a-tool/customize_an_llm_tool.md

```diff
@@ -19,8 +19,17 @@ Here we use [an existing tool package](https://github.com/microsoft/promptflow/t
 
 
     @tool
-    def my_tool(connection: CustomConnection, prompt: PromptTemplate, **kwargs) -> str:
-        # Customize your own code to use the connection and prompt here.
+    def my_tool(
+        connection: CustomConnection,
+        endpoint_name: str,
+        api: str,
+        temperature: float,
+        prompt: PromptTemplate,
+        **kwargs
+    ) -> str:
+        # Replace with your tool code, customise your own code to handle and use the prompt here.
```

Comment:
Why do we need to update the example by adding more param? Is there any user ask or issue?